fontforgeexe/metricsview.c

```diff
@@ -5291,7 +5291,7 @@ MetricsView *MetricsViewCreate(FontView *fv,SplineChar *sc,BDFFont *bdf) {
 
     for ( cnt=0; cnt<mv->clen; ++cnt ) {
         int cp = mv->chars[cnt]->unicodeenc;
-        if ( cp != -1 && strchr("#[]/\\", cp) == NULL)
+        if ( cp != -1 && u_strchr((unichar_t[]){'#', '[', ']', '/', '\\'}, cp) == NULL )
```

Comment:
This is missing a null terminator, it might also be a bight clearer if you define it as `static const unichar_t*` out of line